dataactvalidator/scripts/loadFile.py

```diff
@@ -99,7 +99,16 @@ def load_country_codes(filename):
     logger.info('{} records inserted to {}'.format(num, table_name))
 
 
-def load_cfda_program(filename):
+def load_cfda_program(base_path):
```

Comment:
If you're going to be removing this from the entire loadFile process, it should not be left in this file. The more important question is: "what is the point of splitting it out?"

Reply:
We want to display our clients a clear view of data loads, so we needed to separate out the functions into individual data loads that can be called separately by initialize.py. @kwhickey